tests/e2e/utils/network_utils.go

```diff
@@ -22,10 +22,18 @@ import (
 	"strconv"
 
 	aznetwork "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2017-09-01/network"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-07-01/network"
```

Comment:
could we use the above 2017-09-01 here? we can update the API version in another separate PR

Reply:
yes sir sure